CIP-1694/README.md

```diff
@@ -1,530 +1,874 @@
+* `No Confidence`
 
+  If an Ada holder delegates to `No Confidence`, then their stake is counted as
+  a **no** vote on every governance action apart from a motion of no confidence (action 1).
+  This also signals that they have no confidence in the existing constitutional committee.
+
+  The effect of delegating to `No Confidence` on chain is that this stake *will* be considered to be
+  a part of the active voting stake. It will count as a `Yes` vote on every `No Confidence`
+  action and a `No` vote on every other action.
+  It also serves as a directly auditable measure of the confidence of the constitutional
+  committee at all times.
 
-* **SPOs**<br/>
-  The SPO vote threshold which must be met as a percentage of the stake held by all stake pools. The SPO vote is only considered if the AVST threshold is :x: or the AVST is below the AVST threshold.
-
-| Governance Action Type                               | Constitutional Committee | DReps    | AVST     | AVST Fallback | SPOs     |
-| ---                                                  | :---:                    | ---      | ---      | ---           | ---      |
-| 1. Motion of no-confidence                           | :x:                      | $P_1$    | $Q_1$    | None          | $R_1$    |
-| 2(a). New Committee/quorum (_normal state_)          | :heavy_check_mark:       | $P_{2a}$ | $Q_{2a}$ | SPO Vote      | $R_{2a}$ |
-| 2(b). New Committee/quorum (_state of no-confidence_)| :x:                      | $P_{2b}$ | $Q_{2b}$ | None          | $R_{2b}$ |
-| 3. Update to the Constitution                        | :heavy_check_mark:       | $P_3$    | $Q_3$    | SPO Vote      | $R_3$    |
-| 4. Hard-Fork initiation                              | :heavy_check_mark:       | $P_4$    | 0        | -             | $R_4$    |
-| 5. Protocol parameter changes                        | :heavy_check_mark:       | $P_5$    | $Q_5$    | SPO Vote      | $R_5$    |
-| 6(a). Treasury withdrawal, $[T_0, T_1)$              | :heavy_check_mark:       | $P_{6a}$ | $Q_{6a}$ | SPO Vote      | $R_{6a}$ |
-| 6(b). Treasury withdrawal, $[T_1, T_2)$              | :heavy_check_mark:       | $P_{6b}$ | $Q_{6b}$ | SPO Vote      | $R_{6b}$ |
-| 6(c). Treasury withdrawal, $[T_2, T_3)$              | :heavy_check_mark:       | $P_{6c}$ | $Q_{6c}$ | SPO Vote      | $R_{6c}$ |
-
-Some of the parameters given in this table ( $P_1$ ... $R_{6c}$, $T_1$ ... $T_3$ ) may be updatable protocol parameters, but others should be hard-coded. This proposal deliberately leaves both this choice and the choice of actual parameter values open for discussion.
 
 > **Note**
-> For all treasury withdrawals, the withdrawal threshold is the **total** amount of Lovelace that is withdrawn by the action, not the amount of any single withdrawal if the action specifies more than one withdrawal.
+> Any Ada holder may register their stake credential as a DRep and delegate to themselves if they wish to actively participate in
```

Comment:
> register their stake credential

*Register a hash of their stake credential? 

Are stake credentials the only credentials being encouraged to be used for basis of dRep IDs?

Reply:
The stake credential is a hash of either a VKey or a script. But I think it is correct to say "register their stake credential".

We will not call the credential used by DRep "stake credentials". The will use the same crypto (ED25519), but folks should not re-use stake keys for this (though we cannot stop it from happening).

Reply:
There isn't any difference between stake & other vkey credentials (see my other comment)